kuma/static/styles/components/wiki/indicators.scss

```diff
@@ -206,14 +269,9 @@ icons for both types of indicators
     @include indicator-icon('\f017'); /* clock */
 }
 
+.esHarmony,
```

Comment:
We need to standardize on how classes are written. For the vast majority of the code base, and pretty much the standard across projects, class names should follow the pattern `descriptive-word` and not the JS variable form of `descriptiveWord`

Therefore this needs to be updated to be `es-harmony`. There are a couple of classes here that does not follow that convention, we can clean this up in a follow on pull request.